extensions/amp-story/1.0/pagination-buttons.js

```diff
@@ -114,9 +120,15 @@ class PaginationButton {
       this.element.querySelector('button')
     );
 
+    /** @private @const {!../../../src/service/localization.LocalizationService} */
+    this.localizationService_ = getLocalizationService(doc);
```

Comment:
Do we need to check/handle if the service is available?
I see some areas of the code that check for this:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/simple-template.js#L87

Reply:
I think the `devAssert` are stripped from the production bundle.